library/src/test/java/com/a99/rxplaces/ExtensionsKtTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.a99.rxplaces
+
+import com.a99.rxplaces.options.AutocompleteType
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.CoreMatchers.not
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Test
+import java.util.Random
+
+class ExtensionsKtTest {
+  @Test
+  fun formatComponents_emptyList() {
```

Comment:
We should improve the test names. When reading those tests, it is not clear what the desired behaviour for the extensions